spec/pom.xml

```diff
@@ -111,6 +123,44 @@
                     <headerFooter>true</headerFooter>
                 </configuration>
             </plugin>
+            <plugin>
+                <groupId>org.codehaus.mojo</groupId>
+                <artifactId>xml-maven-plugin</artifactId>
+                <version>1.0.1</version>
```

Comment:
There is a later patch release available meanwhile (http://www.mojohaus.org/xml-maven-plugin/): `org.codehaus.mojo:xml-maven-plugin:1.0.2`

Reply:
I also saw this version on the plugin homepage. And I tried this version but failed. This version isn't available in Maven Central. Just some beta version. See [here](https://search.maven.org/#search%7Cgav%7C1%7Cg%3A%22org.codehaus.mojo%22%20AND%20a%3A%22xml-maven-plugin%22). Really weird.

Reply:
I literally hate the way Maven Central is managed. You need to obey a million rules to get your stuff accepted there, but if you want to download a dependency it fails due to uploaded crap, incompatible alpha versions, or broken Nexus indices. Sonatype should pass management of Maven Central over to us. ;-)